README.md

```diff
@@ -6,7 +6,8 @@
 
 zchunk is a compressed file format that splits the file into independent chunks.
 This allows you to only download changed chunks when downloading a new version
-of the file, and also makes zchunk files efficient over rsync.
+of the file.  Files can hosted on any web server that supports HTTP ranged
+requests, with no special software required to serve the files.
```

Comment:
The client still does though, yes?

Reply:
Yes, should I add in something to clarify that?

Reply:
I think it would help